packages/plugins/custom-fields/admin/src/components/ColorPicker/ColorPickerInput/index.js

```diff
@@ -0,0 +1,198 @@
+  padding: ${({ theme }) => theme.spaces[1]};
+  min-height: 260px;
+`;
+
+const ColorPickerInput = ({
+  attribute,
+  description,
+  disabled,
+  error,
+  intlLabel,
+  labelAction,
+  name,
+  onChange,
+  required,
+  value,
+}) => {
+  const [showColorPicker, setShowColorPicker] = useState(false);
+  const colorPickerButtonRef = useRef();
+  const { formatMessage } = useIntl();
+  const color = value || '#ffffff';
+  const styleUppercase = { textTransform: 'uppercase' };
+
+  const handleBlur = (e) => {
+    e.preventDefault();
+
+    if (!e.currentTarget.contains(e.relatedTarget)) {
+      setShowColorPicker(false);
+    }
+  };
+
+  return (
+    <Field
+      name={name}
+      id={name}
+      // GenericInput calls formatMessage and returns a string for the error
+      error={error}
+      hint={description && formatMessage(description)}
+    >
+      <Stack spacing={1}>
+        <Flex>
+          <FieldLabel action={labelAction} required={required}>
+            {formatMessage(intlLabel)}
+          </FieldLabel>
+        </Flex>
+        <ColorPickerToggle
+          ref={colorPickerButtonRef}
+          aria-label="Color picker toggle"
+          aria-controls="color-picker-value"
+          aria-haspopup="dialog"
+          aria-expanded={showColorPicker}
+          aria-disabled={disabled}
+          disabled={disabled}
+          onClick={() => setShowColorPicker(!showColorPicker)}
+        >
+          <Flex>
+            <ColorPreview color={color} />
+            <Typography
+              style={styleUppercase}
+              textColor={value ? null : 'neutral600'}
+              variant="omega"
+            >
+              {color}
+            </Typography>
+          </Flex>
+          <CarretDown aria-hidden />
+        </ColorPickerToggle>
+        {showColorPicker && (
+          <ColorPickerPopover
+            onBlur={handleBlur}
+            role="dialog"
+            source={colorPickerButtonRef}
+            spacing={4}
+          >
+            <FocusTrap onEscape={() => setShowColorPicker(false)}>
+              <ColorPicker
+                color={color}
+                onChange={(hexValue) =>
+                  onChange({ target: { name, value: hexValue, type: attribute.type } })
+                }
+              />
+
+              <Flex paddingTop={3} paddingLeft={4} justifyContent="flex-end">
+                <Box paddingRight={2}>
+                  <Typography variant="omega" as="label" textColor="neutral600">
+                    {formatMessage({
+                      id: getTrad('color-picker.input.format'),
+                      defaultMessage: 'HEX',
+                    })}
+                  </Typography>
+                </Box>
+                <FieldInput
+                  id="color-picker-value"
+                  aria-label="Color picker input"
+                  style={styleUppercase}
+                  value={value}
+                  placeholder="#FFFFFF"
```

Comment:
I think it may be confusing to see white by default. Maybe we could think of a differnet "no color" state